employee-rostering-backend/pom.xml

```diff
@@ -364,96 +243,6 @@
-      </dependencies>
-      <build>
-        <pluginManagement>
-          <plugins>
-            <plugin>
-              <groupId>org.apache.maven.plugins</groupId>
-              <artifactId>maven-surefire-plugin</artifactId>
-              <configuration>
-                <systemPropertyVariables>
-                  <jacoco-agent.destfile>target/jacoco.exec</jacoco-agent.destfile>
-                </systemPropertyVariables>
-              </configuration>
-            </plugin>
-          </plugins>
-        </pluginManagement>
-        <plugins>
-          <plugin>
-            <groupId>org.jacoco</groupId>
-            <artifactId>jacoco-maven-plugin</artifactId>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-    <profile>
-      <id>sonar</id>
-      <properties>
-        <sonar.projectKey>kiegroup_optaweb-vehicle-routing</sonar.projectKey>
-        <sonar.organization>kiegroup</sonar.organization>
-        <sonar.host.url>https://sonarcloud.io/</sonar.host.url>
-        <!--suppress UnresolvedMavenProperty -->
-        <sonar.login>${env.SONARCLOUD_TOKEN}</sonar.login>
-      </properties>
-      <build>
-        <plugins>
-          <plugin>
-            <groupId>org.sonarsource.scanner.maven</groupId>
-            <artifactId>sonar-maven-plugin</artifactId>
-          </plugin>
-        </plugins>
-      </build>
-    </profile>
-    <profile>
-      <!-- Skip Sonar plugin execution if SONARCLOUD_TOKEN is not set. -->
-      <id>skipSonar</id>
-      <activation>
-        <property>
-          <name>!env.SONARCLOUD_TOKEN</name>
-        </property>
-      </activation>
-      <properties>
-        <sonar.skip>true</sonar.skip>
-      </properties>
-    </profile>
-    <profile>
-      <!-- Don't set sonar.pullrequest properties when the Travis build is not triggered by PR (for example by push
-           to branch or by cron). Sonar plugin would fail because the content of these properties would be invalid. -->
-      <!-- See https://docs.travis-ci.com/user/environment-variables/ for list of Travis environment variables. -->
-      <id>pullRequest</id>
-      <activation>
-        <property>
-          <name>env.TRAVIS_EVENT_TYPE</name>
-          <value>pull_request</value>
-        </property>
-      </activation>
-      <properties>
-        <!--suppress UnresolvedMavenProperty -->
-        <sonar.pullrequest.base>${env.TRAVIS_BRANCH}</sonar.pullrequest.base>
-        <!--suppress UnresolvedMavenProperty -->
-        <sonar.pullrequest.branch>${env.TRAVIS_PULL_REQUEST_BRANCH}</sonar.pullrequest.branch>
-        <!--suppress UnresolvedMavenProperty -->
-        <sonar.pullrequest.key>${env.TRAVIS_PULL_REQUEST}</sonar.pullrequest.key>
-      </properties>
-    </profile>
   </profiles>
 
-  <repositories>
-    <repository>
-      <id>jboss-public-repository-group</id>
-      <name>JBoss Public Repository Group</name>
-      <url>https://repository.jboss.org/nexus/content/groups/public/</url>
```

Comment:
FYI, you'll need this repository once you change your optaplanner dependency to a snapshot version. JBoss snapshot artifacts are not available in Maven Central repository.

You don't need to declare the repository here if you have `optaweb-employee-rostering` as the parent of this POM (which you currently don't). Its parent is `kie-parent` where this repository is declared, so you'll inherit it from there. If you want to switch the parent from `spring-boot-starter-parent` to `kie-parent` (which is a good idea), follow https://www.baeldung.com/spring-boot-dependency-management-custom-parent.